bot_modules/dump_analyse/index.js

```diff
@@ -3,7 +3,7 @@ const GitHub = require('github-api')
 const ipRangeCheck = require('ip-range-check')
 const mcping = require('mcping-js')
 
-let config = {}
+const config = require('../config_manager/index.js')
```

Comment:
This should use `const { configEditor} = ...` so you don't have to do `config.configEditor`

Reply:
Done